package.json

```diff
@@ -79,7 +79,7 @@
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-react": "^7.34.3",
     "eslint-plugin-react-hooks": "^4.6.2",
-    "prettier": "^3.3.2",
+    "prettier": "^3.3.3",
```

Comment:
@dai-shi when I ran `pnpm i` for the first time, it got 3.3.3 because of the ^

I chose to just update here to make that change more obvious.

I can exclude the lockfile and package.json updates in this PR if that is your preference though. Just figured I'd leave it in and let you decide what to do.

Reply:
I wonder how it happens, but in general I prefer 👇 

> I can exclude the lockfile and package.json updates in this PR if that is your preference though

Reply:
Updated to exclude those changes! :)